### ISSUE TITLE
The precision of prunned resnet50, traning from scratch.

### ISSUE THREAD

**Issue by liushuhit** [liuzechun/MetaPruning#13]
Hi, I referenced the given prunned network of resnet50 of FLOPs 2G（ids = [23, 17, 16, 17, 16, 16, 18, 20, 18, 25, 11, 29, 22, 18, 13, 27, 17, 23, 23, 20]）, and training from scrach on the given code, but i can't get  the accuracy given in the paper. Can you give me some suggesttion about  the parameter configuration and GPU  number and the training method in your training ,thank you very much.

**Comment by pecanjk** [comment]
what is the accuracy of your results?  If training from scratch could not work then the assumptions about structure matters in the original paper is wrong.

**Comment by liuzechun0216** [comment]
> Hi, I referenced the given prunned network of resnet50 of FLOPs 2G（ids = [23, 17, 16, 17, 16, 16, 18, 20, 18, 25, 11, 29, 22, 18, 13, 27, 17, 23, 23, 20]）, and training from scrach on the given code, but i can't get the accuracy given in the paper. Can you give me some suggesttion about the parameter configuration and GPU number and the training method in your training ,thank you very much.

Hi, thanks for your interest in my work, the configuration is batchsize 256, initial learning rate 0.1, weight decay is set to 1e-4 and the total training epoch is 128 epoches. How much accuracy did you get when training resnet from scratch?

**Comment by liuzechun** [comment]
> Hi, I referenced the given prunned network of resnet50 of FLOPs 2G（ids = [23, 17, 16, 17, 16, 16, 18, 20, 18, 25, 11, 29, 22, 18, 13, 27, 17, 23, 23, 20]）, and training from scrach on the given code, but i can't get the accuracy given in the paper. Can you give me some suggesttion about the parameter configuration and GPU number and the training method in your training ,thank you very much.

Hi, thanks for your interest in my work, the configuration is batchsize 256, initial learning rate 0.1, weight decay is set to 1e-4 and the total training epoch is 128 epoches. How much accuracy did you get when training the original resnet from scratch?

**Comment by liuzechun** [comment]
> what is the accuracy of your results? If training from scratch could not work then the assumptions about structure matters in the original paper is wrong.

The accuracy is reported in the paper, which is trained from scratch, the model is also uploaded.
I think the assumptions about structure matters is proved in the paper "Rethinking the Value of Network Pruning", which is not the claim made in our paper. The claim of our paper is that we can use the meta pruning to efficiently train the pruning net and efficiently search for good pruned networks.
